finance/proposal-calls/cycle3/user-dev-support-rse.md

```diff
@@ -0,0 +1,86 @@
+choice of which ones to address will be guided by community input, and the experience
+and expertise of the RSE, under the manager's supervision. To reduce the burden on volunteer
+maintainers, the funded manager will perform a first review where appropriate, to keep the
+effort needed for the sub-package maintainers to a minimum.
+
+The RSE position is distinct from the Software Operational Support Specialist (SOSS)
+position in that the RSE is expected to take on and complete discrete *projects*, rather
+than provide continuous support (e.g., monitoring issues, helping new contributors,
+etc., as was included in the SOSS scope of work).
+
+On the other hand, the RSE shall work on the items listed on [Astropy
+Roadmap](https://github.com/astropy/astropy-project/blob/main/roadmap/roadmap.md), that the
+community has already prioritized, but not yet found the resources to implement.
+
+Example projects:
+* *Restructure the Astropy developer documentation*: The current developer documentation
+  pages have a wealth of content aimed at both new contributors and expert technical
+  engineers, but it is mixed and hard for contributors with different skill levels to
+  easily find the content most appropriate for them. The goal of this project is to
+  restructure the documentation pages to provide easier pathways for potential or
+  existing contributors with different skill levels to find relevant developer
+  documentation they need to make a contribution. Relevant issues:
+  [#11620](https://github.com/astropy/astropy/issues/11620),
+  [#11621](https://github.com/astropy/astropy/issues/11621)
+* *Implement robust benchmark reporting for pull requests* ([drawn from the Astropy
+Roadmap](https://github.com/astropy/astropy-project/blob/main/roadmap/roadmap.md)): This
+  project would enable a way of evaluating the performance impact of pull requests by
+  tracking and benchmarking existing performance of common tasks (see, for example, the
+  old [astropy-benchmarks](https://github.com/astropy/astropy-benchmarks)).
+* *Implement integration testing for core, coordinated, and infrastructure packages*
+([drawn from the Astropy
+  Roadmap](https://github.com/astropy/astropy-project/blob/main/roadmap/roadmap.md)):
+  Ensure that coordinated and infrastructure packages have integration tests that check
+  the joint usage of Astropy and these other packages together, rather than individually
+  (as is often the case now).
+
+
+### Approximate Budget
+- Currency: US $84,000
+- Minimum to be viable: US $64,000 (approx. $60/hour)
+- Maximum: Up to a full time appointment (approx. $160,000)
+
+**Position**: Research Software Engineer
+
+Budget breakdown (nominal):
+- Travel: $8000
+- Conference Registrations: $1000
+- Salary / contractor fees etc.: $75000 (approx. $80/hour)
+- TOTAL = $84000
+
+**Position**: Manager of Research Software Engineer
+
+- Moritz Guenther, funded by sub-award to MIT
+- $23,155.29 for 0.1 FTE (1 day/week for 0.5 years)
```

Comment:
So, no manager after 6 months?

Reply:
I have not heard back from the finance committee if the RSE would be 50 % for 12 months or 100% for 6 months or something in between. That depends on whom we find and how fast the money needs to be spend. I will be funded for the same time span as the RSE.

Reply:
On top Budget says $84,000 under _Currency_, but the breakdown lists $84,000 for the developer + $23,155 for the manager – so the grand total would be around $107,000 (or $160,000-$170,000 for a full-time position)?

Reply:
I'm updating all numbers now that I received a budget from the finance committee.